static/scss/search-page.scss

```diff
@@ -56,6 +56,11 @@
       padding: 0 10px 0 23px;
     }
 
+    .sk-hierarchical-menu-list {
+      max-height: 150px;
+      overflow-y: scroll;
```

Comment:
This still doesn't cause the scrollbars to show up all the time, as far as my testing shows.